src/test/unit/specific_feature_tests/native-edit-registration.js

```diff
@@ -0,0 +1,55 @@
+/* global
+    QUnit,
+    prepareUnitTestModule,
+    skipKeyboardShortcutTests,
+    simulateKeyCombo
+*/
+
+"use strict";
+
+module("native-edits-registering", {setup: prepareUnitTestModule});
+
+// Registration of native edits is almost entirely external module code.
+// Specifically, the 'edited' module.
+// The following tests are not meant to fully cover 'edited'.
+// They are meant to test that 'edited' seems to be working.
+
+QUnit.test("edited is instantiated", function () {
+    QUnit.expect(1);
+
+    var wymeditor = jQuery.wymeditors(0),
+        edited = wymeditor.nativeEditRegistration.edited,
+        Edited = WYMeditor.EXTERNAL_MODULES.Edited;
+    QUnit.ok(edited instanceof Edited, "instanceof");
+});
+
+QUnit.test("'edited' seems to be working", function () {
+    if (skipKeyboardShortcutTests) {
+        QUnit.expect(0);
+        return;
+    }
+    QUnit.expect(1);
+
+    var wymeditor = jQuery.wymeditors(0),
+        thisTest = this,
+        callCount,
+        p;
+
+    thisTest.spy(wymeditor, 'registerModification');
+
+    // on some keyboard events the editor expects real selection
+    wymeditor.html("<p>foo</p>");
+    p = wymeditor.$body().find("p")[0];
+    wymeditor.setCaretIn(p);
+
+    simulateKeyCombo(wymeditor, 'a'); // 1st callback
+    simulateKeyCombo(wymeditor, '3'); // no callback
+    simulateKeyCombo(wymeditor, ' '); // 2nd callback
+    simulateKeyCombo(wymeditor, ' '); // no callback
+    simulateKeyCombo(wymeditor, '.'); // 3rd callback
+
+    callCount = wymeditor.registerModification.callCount;
+    QUnit.strictEqual(callCount, 3, "`registerModification` called 3 times");
+
+    wymeditor.registerModification.restore();
+});
```

Comment:
We can't test content changes in this regard using unit tests because we are only simulating keyboard events. The native actions do not really occur. So there is no content change. Attempting to simulate the content changes will be complex and confusing.

External tests are what we're after, here.

I don't yet start that, so I think we can quite safely merge this in. These tests are enough because we're testing that Edited is operational.

We're trusting that it is doing its part.

It is far better to have further confirmation, in the general `undo_redo.js` tests file, that native and editor modifications are getting undone/redone as expected. But, again, that should be done using external test.